platforms/access-bridge/test/fixtures.ts

```diff
@@ -146,3 +151,96 @@ export const AUTHORIZATION = {
   VALID: 'Bearer valid-authorization',
   INVALID: 'Bearer invalid-authorization',
 };
+
+// Stripe price mock
+export const STRIPE_PRICE: Stripe.Price = {
+  id: 'price_123456789',
+  object: 'price',
+  active: true,
+  billing_scheme: 'per_unit',
+  created: PAST_EXPIRY,
+  currency: 'usd',
+  custom_unit_amount: null,
+  livemode: false,
+  lookup_key: 'mock_lookup_key',
+  metadata: {},
+  nickname: 'Standard Price',
+  product: 'prod_123456789',
+  recurring: {
+    interval: 'month',
+    meter: null,
+    interval_count: 1,
+    usage_type: 'licensed',
+    aggregate_usage: null,
+    trial_period_days: 1,
+  },
+  tax_behavior: 'exclusive',
+  tiers_mode: 'graduated',
+  transform_quantity: {
+    divide_by: 1,
+    round: 'up',
+  },
+  type: 'recurring',
+  unit_amount: 1000,
+  unit_amount_decimal: '10.00',
+};
+
+// Stripe product mock
+export const STRIPE_PRODUCT: StripeProduct = {
```

Comment:
This comment isn't really about this file, but it made me think. I'm really nervous about how we're going to nicely show the huge amount of data that stripe returns. What are your thoughts on it? I worry that we are going to have many of the same problems that Sims was having on the management side (there are just too many features for us to keep up with.)

Part of me want to just use the stripe pricing tables, so it's off of our plate, but that's a big product architecture change. Alternatively, maybe we could think of a slick way to define mappings that could be easily changed or even controlled via app config to let customers deal with it as they want.

Some limitations I see:
* showing a description about what the plan gives you access to and special features (downloads, number of streams, etc.) Do we just say that we use name and description?
* Making sure we show all the recurrence details properly (what are all the possible options from stripe?)
* Showing the right currency and tax for the viewer
* Showing all the details of the free trials (again what are all the possible options?)

Whatever we decide, this is definitely something we need to document clearly.

Reply:
I would say for now we just follow the designs and show the minimum required stuff that is necessary to make a subscription. 

Stripe pricing tables look promising, but wouldn't change it now in sake of the spotlight and it may affect SIMS side as well. 
Also, not seeing the downside of returning the entire Stripe product, as the customers can add features by their preferences. Isn't this supposed to serve only as a starting point? Why do we wanna handle cases we don't know will exist..

We can document what we support and of course it can always be extended as we add support to more features.